core/src/test/scala/unit/kafka/log/LogCleanerManagerTest.scala

```diff
@@ -413,7 +413,7 @@ class LogCleanerManagerTest extends Logging {
 
     // updateCheckpoints should remove the topicPartition data in the logDir
     cleanerManager.updateCheckpoints(logDir, partitionToRemove = Option(topicPartition))
-    assertTrue(cleanerManager.allCleanerCheckpoints.get(topicPartition).isEmpty)
+    assertTrue(!cleanerManager.allCleanerCheckpoints.contains(topicPartition))
```

Comment:
Would `assertFalse(cleanerManager.allCleanerCheckpoints.contains(topicPartition))` be even more readable?

Reply:
done.

Reply:
There's a few more `assertTrue(!` below, can we update them too?